install_config/registry/extended_registry_configuration.adoc

```diff
@@ -581,6 +586,35 @@ be safe to enable its support in the registry. See the
 xref:docker-registry-configuration-reference-middleware[middleware
 configuration reference] above for particular option.
 
+[[docker-registry-configuration-reference-openshift]]
```

Comment:
On its own, it would make sense. However, each section at this level is named the same as the configuration section. I would rather hold onto the convention here.